WordPress/Classes/Extensions/NSAttributedString+Helpers.swift

```diff
@@ -33,8 +33,7 @@ public extension NSAttributedString {
             if
                 displayAnimatedGifs,
                 let animatedImage = image as? AnimatedImage,
-                animatedImage.gifData != nil
-            {
+                animatedImage.gifData != nil {
```

Comment:
Moving up from version 0.54.0 brought up this and other `opening_brace` violations.

Might be related to the rule having been [rewritten in version 0.55.0](https://github.com/realm/SwiftLint/releases/tag/0.55.0) and then [updated in version 0.57.0](https://github.com/realm/SwiftLint/releases/tag/0.57.0).